netZooPy/lioness/lioness.py

```diff
@@ -48,12 +52,12 @@ class Lioness(Panda):
         TF, Gene and Motif order is identical to the panda output file.
 
     Authors: 
-        cychen, davidvi
+        cychen, davidvi, dcolinmorgan
 
     Reference:
         Kuijjer, Marieke Lydia, et al. "Estimating sample-specific regulatory networks." Iscience 14 (2019): 226-240.
     """
-    def __init__(self, obj, computing='cpu', precision='double',start=1, end=None, save_dir='lioness_output', save_fmt='npy'):
+    def __init__(self, obj, computing='cpu', precision='double',ncores=1,start=1, end=None, save_dir='lioness_output', save_fmt='npy'):
```

Comment:
Hi @dcolinmorgan, why is there an `ncores` parameter here? If this is related to #156 , it is better to have on PR per feature.